src/api/main.py

```diff
@@ -103,6 +99,8 @@ async def lifespan(app: fastapi.FastAPI):
         # Create index and upload the documents only if index does not exist.
         logger.info(f"Creating index {os.getenv('AZURE_AI_SEARCH_INDEX_NAME')}.")
         await rag.ensure_index_created(vector_index_dimensions=embed_dimensions if embed_dimensions else 100)
+    else:
+        logger.info("The RAG search will not be used.")
 
     globals["project"] = project
     globals["chat"] = chat
```

Comment:
for globals we can use the app.state, please check from agents template

Reply:
Fixed.